sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -237,6 +255,7 @@ def create(value, timestamp_micros, windows, pane_info=PANE_INFO_UNKNOWN):
 
 
 try:
+  # FIXME: for review: why not add this as a class attribute?
```

Comment:
See comment below. (This is an artifact of trying to make Cython an /optional/ dependency.)